AEPMessaging/Sources/Messaging.swift

```diff
@@ -198,6 +203,113 @@ public class Messaging: NSObject, Extension {
         return true
     }
 
+    // MARK: - Event Handers
+
+    /// Processes the events in the event queue in the order they were received.
+    ///
+    /// A valid `Configuration` and `EdgeIdentity` shared state is required for processing events.
+    ///
+    /// - Parameters:
+    ///   - event: An `Event` to be processed
+    func handleProcessEvent(_ event: Event) {
```

Comment:
this function isn't actually new - it's moved from a different location in the file